src/ec/packages/ec-component-text-input/text-input.test.js

```diff
@@ -0,0 +1,71 @@
+import path from 'path';
+import { renderTwigFile } from '@ecl-twig/test-utils';
+
+describe('EC - Text input', () => {
+  const template = path.resolve(__dirname, './text-input.html.twig');
+
+  describe('Default', () => {
+    test('renders correctly', done => {
+      expect.assertions(1);
+
+      renderTwigFile(
+        template,
+        {
+          label: 'Label',
```

Comment:
why don't you use the data from the specs?

Reply:
it's not the same structure